app/models/item.rb

```diff
@@ -77,8 +77,16 @@ def deleted?
     deleted_at != nil
   end
 
-  def total_value
-    current_quantity * value
+  def past_version_value(time)
+    past_version = versions.sort_by(&:created_at).reverse
+                           .find { |v| v.created_at <= time }.try(:reify)
```

Comment:
If it's ok lets use the `&.` opperand here.
http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

Reply:
@tekHudson I'll take it. Btw what is your thoughts on that UI issue regarding the total + pagination.

Reply:
Looks like we need to use a footer callback https://datatables.net/examples/advanced_init/footer_callback.html

Reply:
I validated we have this issue on live with the Value by County report

Reply:
Let's merge it after my fix and create a new ticket

Reply:
Sooooo...... turns out the `&.` operand wasn't added until 2.3.0 🤦‍♂️  So I'm gonna merge this and make 3 new tickets. One to move use to 2.3 or better 2.4.2, one to replace `.try()` with `&.` and another to add the footer callback.